framework/test/Volo.Abp.Caching.Tests/Volo/Abp/Caching/PersonCacheItem.cs

```diff
@@ -17,7 +17,14 @@ public PersonCacheItem(string name)
             Name = name;
         }
     }
+
+    public class DummyObjectAsCacheKey
```

Comment:
Override ToString and return a merged value of the properties. Otherwise all keys will be same (you know default ToString returns the class name - same for all objects)

Reply:
Yeah, i missed the part about default ToString method of  the object. 
Although, I wanted to ask; does the generic object's ToString need to be overriden? Wouldn't it cause all the object's ToString method modified and should be declared in some sort of extension class (I think it already exists somewhere)?

Or do you think the complex object to be used as key should be marked with an interface like below?
```javascript
public class DummyObjectAsCacheKey: ICanBeCacheKey
```
I thought handling it inside the 
```javascript
protected virtual string NormalizeKey(TCacheKey key)
```
is more effective (but might be less efficient since checking type of key in every call).

I'll be sending new PR and let me know if I can improve.